test/contracts/supplementary/lib/exchangeIssuanceLibraryMock.spec.ts

```diff
@@ -130,12 +130,61 @@ contract('ExchangeIssuanceLibraryMock', accounts => {
     });
 
     describe('when Set quantities is zero', async () => {
-      before(async () => {
+      beforeEach(async () => {
```

Comment:
It's not explicitly in this PR but we don't have a naturalUnit multiple test for the `validateQuantity` function. We should add that while we're doing this.